playbooks/azure/openshift-cluster/files/get-node-logs

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -x
+
+LOG_DIR=$(mktemp -d)
+
+trap "{ rm -rf $LOG_DIR ; exit 1; }" INT TERM ERR
+
+SYSTEMD_SERVICES=("dnsmasq NetworkManager atomic-openshift-node origin-node")
+
+since_docker="24h"
+since_systemd="24 hour ago"
+
+# Dump systemd services logs
+for name in ${SYSTEMD_SERVICES} ; do
+  dump_file_path=${LOG_DIR}/${name}.log
+  journalctl -u ${name}.service --since "${since_systemd}" > $dump_file_path
+done
+
+# Dump command output
+ip a > ${LOG_DIR}/ip_a.log
+netstat -antu > ${LOG_DIR}/netstat_antu.log
+dmesg > ${LOG_DIR}/dmesg.log
+route -n > ${LOG_DIR}/route_n.log
+ss -ntpl > ${LOG_DIR}/ss_ntpl.log
+cat /etc/resolv.conf > ${LOG_DIR}/resolve_conf.log
+df -h > ${LOG_DIR}/df_h.log
+vmstat 2 20 > ${LOG_DIR}/vmstat_2_20.log
```

Comment:
can leave this in for now but it's slow and I hope prometheus will do this for us one day ;)